migration.md

```diff
@@ -46,6 +46,14 @@ distribution of Splunk Distribution of OpenTelemetry Java Instrumentation:
    setting.
 4. In your application startup script, replace `-javaagent:./signalfx-tracing.jar`
    with `-javaagent:/path/to/splunk-otel-javaagent-all.jar`.
+5. If you manually instrumented any code with an OpenTracing tracer, expose
+   the OpenTelemetry tracer as an implementation of an OpenTracing tracer with
+   the OpenTracing Shim. For more information, see
+   [OpenTelemetry - OpenTracing Shim](https://github.com/open-telemetry/opentelemetry-java/tree/master/opentracing-shim).
```

Comment:
Is there anything special you need to do to enable the OT Shim when running with the java agent? @iNikem ?

Reply:
Does [this PR](https://github.com/signalfx/tracing-examples/pull/60/files) answer your question?

Reply:
If that code works, then yes!  😉 
I thought that it would probably "just work" since the shim only uses the API. Thanks!